app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -259,9 +266,27 @@ private fun onBindCustomHomeTile(uiLifecycleCancelJob: Job, holder: TileViewHold
         }
 
         // Wait for both to complete so we can animate them together.
-        iconView.setImageBitmap(screenshot.await()) // TODO: if null, provide placeholder.
+        val screenshotToRound = screenshot.await() // TODO: if null, provide placeholder.
+        val cornerRadius = 8.toFloat()
```

Comment:
Numbers defined in code are pixel values but we should be specifying them in DP so the changes look consistent across devices. The easiest way to do this is define the value in dimens.xml and access it with `context.getDimensionPixelSize`.

In this case, you can use the same corner radius we define for the home tiles themselves.

Please grab all the numbers from resources!

---

No longer necessary but you can create float literals with `8f`.